api/README.md

```diff
@@ -102,6 +102,8 @@ A note about API versions below 1, such as 0.1.0. These versions are under devel
 
 * 0.2.0 - [API Specification](./v1/gateway/0.2.0.yaml) | [API Documentation](./v1/gateway/0.2.0.html)
 
+* 0.3.0 - [API Specification](./v1/gateway/0.3.0.yaml) | [API Documentation](./v1/gateway/0.3.0.html)
+
 [Gateway Service API - Current](./../gateway/gateway-service-api.yaml)
 
 ### [AnyQuiz Service API](#anyquzi-service-api)
```

Comment:
typo #anyquzi- in link

Reply:
Great catch. I'll fix this and add a commit.